### ISSUE TITLE
The 0.8.0 release keeps milestone dependencies (Important!)

### ISSUE THREAD

**Issue by tzolov** [micrometer-metrics/prometheus-rsocket-proxy#27]
The 0.8.0  release (e.g. `io.micrometer.prometheus:prometheus-rsocket-spring:0.8.0`) has unreleased (e.g. milestone) dependencies on `io.rsocket:rsocket-core:jar:1.0.0-RC2`!

```
Failed to collect dependencies at 
io.micrometer.prometheus:prometheus-rsocket-spring:jar:0.8.0 -> 
io.micrometer.prometheus:prometheus-rsocket-client:jar:0.8.0 -> 
io.rsocket:rsocket-core:jar:1.0.0-RC2: 

Failed to read artifact descriptor for io.rsocket:rsocket-core:jar:1.0.0-RC2
```

To use the 0.8.0 release now you have to add spring milestone maven repo. This is likely to be a blocker for the forthcoming Spring Cloud Stream App Starters  release!

**Comment by sabbyanandan** [comment]
@tzolov: FWIW, the RSocket 1.0 GA will be shipping soon, so Spring Framework [5.2 GA](https://github.com/spring-projects/spring-framework/milestone/216) can include it.

**Comment by shakuzen** [comment]
Yes, I realized this while I was finalizing the 0.8.0 release. Unfortunately, there isn't anything we can do until RSocket 1.0 GA is released.

Note that RSocket milestone/RC artifacts are published in Maven Central, but Reactor milestone artifacts are not - like `reactor-bom` version `Dysprosium-M3`. This is why the subprojects declare the spring milestone repository in their builds.

**Comment by tzolov** [comment]
@sabbyanandan, the `spring-framework 5.2 GA` has been released with the [io.rsocket:rsocket-bom:1.0.0-RC5](https://github.com/spring-projects/spring-framework/blob/v5.2.0.RELEASE/build.gradle#L49) dependency. 

@shakuzen ,  would the `rsocket-bom:1.0.0-RC5` help resolve this issue as well?

**Comment by shakuzen** [comment]
Assuming this issue is more about the milestone repository that was required, it will be resolved by #28.

**Comment by jkschneider** [comment]
With the 1.0.0 release, depending on RSocket 1.0.0 GA
